exercises/practice/simple-linked-list/SimpleLinkedListTests.cs

```diff
@@ -1,83 +1,117 @@
+using System;
 using System.Linq;
+using System.Collections.Generic;
 using Xunit;
 
 public class SimpleLinkedListTests
 {
-    [Fact]
-    public void Single_item_list_value()
+    [Fact(Skip = "Remove this Skip property to run this test")] 
```

Comment:
The first test is not skipped

```suggestion
    [Fact] 
```